_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp

```diff
@@ -461,4 +474,21 @@ uint32_t DDI_VA::ConvertRateControlMFX2VAAPI(mfxU16 rateControl, bool bSWBRC)
     return 0;
 }
 
+uint32_t DDI_VA::ConvertSliceStructureVAAPIToMFX(uint32_t structure)
+{
+    switch (structure)
+    {
+        case VA_ENC_SLICE_STRUCTURE_POWER_OF_TWO_ROWS:
+            return uint32_t(1);
```

Comment:
Please avoid usage of 'magic numbers'

Reply:
Agree, Ideally we need to re-use approach from https://github.com/Intel-Media-SDK/MediaSDK/pull/1814/files . And ideally we need to change this hardcoded 1, 2,3, etc across all places

Reply:
> Please avoid usage of 'magic numbers'

Done.